src/vs/workbench/parts/debug/common/debugModel.ts

```diff
@@ -414,6 +424,14 @@ export class Model extends ee.EventEmitter implements debug.IModel {
 		this.emit(debug.ModelEvents.CALLSTACK_UPDATED);
 	}
 
+	public continueThreads(): void {
```

Comment:
Yes, `clearThreads` is already confusing enough. Explicit `continueThreads` is fine imho.